website/snippets/available-enterprise-only.md

```diff
@@ -0,0 +1,5 @@
+:::info Limited to dbt Cloud Enterprise
```

Comment:
If we switch to partials, rename this file to
> website/snippets/_available-enterprise-only.md

Reply:
addressed in https://github.com/dbt-labs/docs.getdbt.com/pull/3616